src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -536,6 +563,7 @@ MTAnalysis >> generateResults [
 	| tests |
 	mutantResults := OrderedCollection new.
 	tests := testFilter filterTests: testCases.
+	self filteredTestsFrom: (testCases copyWithoutAll: tests).
```

Comment:
May I propose to rename `tests` into `filteredTests`? :)

Reply:
Indeed !